acmw-db-app/pages/api/meeting/[pid].js

```diff
@@ -27,6 +16,25 @@ async function averageAttendance () {
     return data.rows[0]["avg"] ? data.rows[0]["avg"] : 0;
 }
 
+// GET /api/meeting/student-attendance
+async function studentAttendance(id) {
+	const [fall, spring] = currentAcademicYear();
+	const numMeetings = await pgQuery(`
+		SELECT m.semester, COUNT(m.id) 
+		FROM meeting m INNER JOIN meeting_student ms ON m.id = ms.meeting_id
+		WHERE ms.student_id = '${id}' AND (m.semester='${fall}' OR m.semester='${spring}')
+		GROUP BY m.semester;
+	`);
+	return numMeetings.rows;
+}
+
+// GET /api/meeting/account-attendance
+async function accountAttendance(email) {
+	const data = await pgQuery(`SELECT s.id FROM student s INNER JOIN account a on s.id = a.student_id WHERE a.email = '${email}'`);
+	if(data.rowCount === 0) throw (`No student associated with account ${email}`);
```

Comment:
I really like bangs like `if(!data.rowCount)`

Reply:
k will change

Reply:
okay. i didn't want to be annoying on your other PR but BANG BANG BANG BANG BANG :) !